pkg/urbit/jets/e/ripe.c

```diff
@@ -17,15 +17,7 @@
     u3r_bytes(0, wid, (void*)dat_y, dat);
 
     const EVP_MD* rip_u = EVP_ripemd160(); // ripem algorithm
-    static EVP_MD_CTX* con_u = NULL;       // context
-
-    /* build library context object
-       we do this once (and only once)
-    */
-
-    if (NULL == con_u) {
-      con_u = EVP_MD_CTX_create();
-    }
+    EVP_MD_CTX* con_u = EVP_MD_CTX_create();
```

Comment:
This is a great catch. It was added in #1020, and I'm embarrassed to say that I approved it ...

With this resolved, have you confirmed that there are no longer any "persistent" OpenSSL allocations that would require special de-initialization on exit? (And therefore, special accommodations in crash recovery?)

Reply:
Yes. I hooked the allocator and ensured that the number of outstanding ssl allocations returned to zero on a whole +test.

(Bisecting with that code with +test was how I zeroed in on this being the problem which was causing the crashing.)